### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by ilyes319** [ACEsuit/mace#216] [Announcements]
## This Release Introduces:
1. Committee of models in the ASE calculator.
2. Active learning for MD in the ASE script.
3. Implementation of the Huber loss function.
4. Can use dipole model within the ASE calculator.
5. A LAMMPS wrapper for executing LAMMPS MD simulations.
6. The matscipy neighbor list set as the default.
7. Gaussian radial basis functions.
8. Element-dependent radial basis functions using a MLP on the radial components and node features of the sender and receiver.
9. MACE descriptors from the ASE calculator.
10. A universal pretrained MACE model on the Materials Project, available as an ASE calculator.

## Major Interface Changes:
1. The scripts have been moved to `mace/cli/` from the previous `scripts` directory to facilitate access.
2. The `mace_run_train` command is now available if MACE is installed via pip.

## What's Changed
* merge readme edits to develop by @davkovacs in https://github.com/ACEsuit/mace/pull/76
* Require stress in shape (3,3) or (6,) by @LarsSchaaf in https://github.com/ACEsuit/mace/pull/77
* Committee error by @LarsSchaaf in https://github.com/ACEsuit/mace/pull/78
* Doc update by @ilyes319 in https://github.com/ACEsuit/mace/pull/85
* no gradient with respect to model params in eval by @davkovacs in https://github.com/ACEsuit/mace/pull/83
* Update active_learning_md.py by @sandipde in https://github.com/ACEsuit/mace/pull/89
* Update utils.py by @sandipde in https://github.com/ACEsuit/mace/pull/91
* Huberloss by @ilyes319 in https://github.com/ACEsuit/mace/pull/96
* not break if exponential LR by @bigd4 in https://github.com/ACEsuit/mace/pull/98
* hopefully limit unnecessary memory use. by @wcwitt in https://github.com/ACEsuit/mace/pull/103
* ASE calculator refactor and jit dipole models by @davkovacs in https://github.com/ACEsuit/mace/pull/95
* Fix unweighted EnergyForcesLoss by @chaitjo in https://github.com/ACEsuit/mace/pull/100
* Lammps wrapper by @wcwitt in https://github.com/ACEsuit/mace/pull/108
* Update Installation by @ilyes319 in https://github.com/ACEsuit/mace/pull/129
* matscipy neighbour list as default by @ilyes319 in https://github.com/ACEsuit/mace/pull/116
* Gaussian radial basis and element dependent radial by @ilyes319 in https://github.com/ACEsuit/mace/pull/131
* MACE descriptors by @ilyes319 in https://github.com/ACEsuit/mace/pull/144
* Add readme section 'Pretrained Universal MACE Checkpoints' by @janosh in https://github.com/ACEsuit/mace/pull/188
* fix table of content readme by @ilyes319 in https://github.com/ACEsuit/mace/pull/190
* Scripts entrypoints by @bernstei in https://github.com/ACEsuit/mace/pull/178
* add foundations model calculators by @ilyes319 in https://github.com/ACEsuit/mace/pull/204
* add deprecated model_path arg to asse calculator by @davkovacs in https://github.com/ACEsuit/mace/pull/209
* MACE 0.3.0 release by @ilyes319 in https://github.com/ACEsuit/mace/pull/142

## New Contributors
* @LarsSchaaf made their first contribution in https://github.com/ACEsuit/mace/pull/77
* @sandipde made their first contribution in https://github.com/ACEsuit/mace/pull/89
* @bigd4 made their first contribution in https://github.com/ACEsuit/mace/pull/98
* @chaitjo made their first contribution in https://github.com/ACEsuit/mace/pull/100

**Full Changelog**: https://github.com/ACEsuit/mace/compare/v0.2.0...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ACEsuit/mace/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by chaitjo** [comment]
Congrats! I'm curious as to how impactful was switching to gaussian RBF vs. the previous RBF function in MACE?
